.gitattributes

```diff
@@ -2,5 +2,8 @@ cassettes/* merge=binary linguist-generated=true
 factory_data/* merge=binary linguist-generated=true
 yarn.lock merge=binary linguist-generated=true
 frontends/api/src/generated/* merge=binary linguist-generated=true
+frontends/api/src/generated/**/* merge=binary linguist-generated=true
+frontends/api/src/mitxonline/generated/* merge=binary linguist-generated=true
+frontends/api/src/mitxonline/generated/**/* merge=binary linguist-generated=true
```

Comment:
These files are generated (manually, for now) based on the openapi spec in https://github.com/mitodl/mitxonline/pull/2549

Two lines (`*`, and `**/*`) are necessary as explained in https://stackoverflow.com/a/62564526/2747370